Resideo_eshopping/resideo_eshopping/lib/Screens/productdetail.dart

```diff
@@ -0,0 +1,114 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:resideo_eshopping/model/eshopping_model.dart';
```

Comment:
Change file name to maintain naming conventions (lower case seperated by under score)
ex: product_detail.dart